### PR TITLE
introduce elixir formatter

### PR DESCRIPTION
**Pull request by stephenmoloney**
## Description

- Introduce elixir formatter for code standardization

- Not sure what the best settings for .formatter.exs are - any thoughts?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,12 @@ matrix:
 notifications:
   email: false
 script:
-  - mix compile --warnings-as-errors
+  - >
```
Comment (doomspork) [inline .travis.yml]: @yordis don't we want something like this for Guardian?

**Comment by doomspork** [comment]
@stephenmoloney could you rebase this PR?

File: .travis.yml (hunk 1 above)
Comment (yordis) [reply .travis.yml]: That was my intent. I am waiting for you to respond about the Elixir version so I replicate the same setup with more features on it as I did on ueberauth

**Comment by stephenmoloney** [comment]
Yea sure, gonna rebase this tomorrow...

**Comment by stephenmoloney** [comment]
rebased, made a minor change to `.formatter.exs` too. Thanks for review.

**Pull request merged**